packages/qdrant-loader/src/qdrant_loader/connectors/git/connector.py

```
@@ -386,11 +388,13 @@ def _process_file(self, file_path: str) -> Document:
             )
             raise
 
-    async def get_documents(self) -> list[Document]:
-        """Get all documents from the repository.
+    async def stream_documents(
+        self, since: datetime | None = None
```

Ruff (0.15.15)

[warning] Unused method argument: `since`

(ARG002)